components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/JSON-validator.js

```diff
@@ -209,15 +209,16 @@ define(['require', 'log', 'jquery', 'lodash', 'designViewUtils'],
             }
         };
 
-        /**
-         * @function Validates a given stream or a table annotation
+		/**
+         * @function Validates a given element
          * @param element element json
-         * @param type stream or table
+         * @param type stream or table or window or trigger or function
          * @param doNotShowErrorMessages If true error messages will not be shown as alerts. Only the validity will be
          * returned
          * @returns {boolean} validity of the json
          */
-        JSONValidator.prototype.validateStreamOrTable = function (element, type, doNotShowErrorMessages) {
+        JSONValidator.prototype.validateForElementName = function (element, type,
+            doNotShowErrorMessages) {
```

Comment:
why new line ?

Reply:
Fixed in 97aff2523a4468759c33ce94e34dd1c88ea092af